source/ranges.tex

```diff
@@ -3202,6 +3210,428 @@
+\end{itemdescr}
+
+\indexlibrarymember{operator++}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator++();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+++@\exposid{current_}@;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator++}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator++(int);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto tmp = *this;
+++*this;
+return tmp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator--}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator--();
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t},
+\tcode{\exposid{bound_} > 0}.
+
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+--current_;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator--}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@ operator--(int);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+auto tmp = *this;
+--*this;
+return tmp;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator+=}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator+=(difference_type n);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t},
+$\exposid{bound_} + \tcode{n} \ge 0$.
+
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+@\exposid{current_}@ += n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator-=}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr @\exposid{iterator}@& operator-=(difference_type n);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\expects
+If \tcode{Bound} is not \tcode{unreachable_sentinel_t},
+$\exposid{bound_} - \tcode{n} \ge 0$.
+
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+current_ -= n;
+return *this;
+\end{codeblock}
+\end{itemdescr}
+
+\indexlibrarymember{operator[]}{repeat_view::iterator}%
+\begin{itemdecl}
+constexpr const W & operator[](difference_type n) const noexcept;
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to: \tcode{return *(*this + n);}
+\end{itemdescr}
+
+\indexlibrarymember{operator==}{repeat_view::iterator}%
+\begin{itemdecl}
+friend constexpr bool operator==(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to \tcode{x.\exposid{current_} == y.\exposid{current_}}.
+\end{itemdescr}
+
+\indexlibrarymember{operator<=>}{repeat_view::iterator}%
+\begin{itemdecl}
+friend constexpr auto operator<=>(const @\exposid{iterator}@& x, const @\exposid{iterator}@& y);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to \tcode{x.\exposid{current_} <=> y.\exposid{current_}}.
```

Comment:
Error in the paper.
```suggestion
Equivalent to \tcode{return x.\exposid{current_} <=> y.\exposid{current_};}.
```

Reply:
see above.

Also, if we add "return", we also need a colon before it.

Reply:
Added the colons.